apm-webapp/src/main/java/org/apache/skywalking/oap/server/webapp/ApplicationStartUp.java

```diff
@@ -16,68 +16,77 @@
-    public static void main(String[] args) throws Exception {
-        final Yaml yaml = new Yaml(
-            new EnvScalarConstructor(
-                new TypeDescription(Configuration.class),
-                Collections.emptyList(),
-                new LoaderOptions()));
-        yaml.addImplicitResolver(ENV_TAG, ENV_FORMAT, "$");
-
-        final Configuration configuration = yaml.loadAs(
-            ApplicationStartUp.class.getResourceAsStream("/application.yml"),
-            Configuration.class);
-
-        final int port = configuration.port();
-        final String[] oapServices = configuration.oapServices();
-
-        final HttpService indexPage =
-            HttpFile
-                .of(ApplicationStartUp.class.getClassLoader(), "/public/index.html")
-                .asService();
-        final HttpService zipkinIndexPage =
-            HttpFile
-                .of(ApplicationStartUp.class.getClassLoader(), "/zipkin-lens/index.html")
-                .asService();
-
-        final ZipkinProxyService zipkin = new ZipkinProxyService(configuration.zipkinServices());
-
-        Server
-            .builder()
-            .port(port, SessionProtocol.HTTP)
-            .service("/graphql", new OapProxyService(oapServices))
-            .service("/internal/l7check", HealthCheckService.of())
-            .service("/zipkin/config.json", zipkin)
-            .serviceUnder("/zipkin/api", zipkin)
-            .serviceUnder("/zipkin",
-                FileService.of(
-                    ApplicationStartUp.class.getClassLoader(),
-                    "/zipkin-lens")
-                    .orElse(zipkinIndexPage))
-            .serviceUnder("/",
-                FileService.of(
-                    ApplicationStartUp.class.getClassLoader(),
-                    "/public")
-                    .orElse(indexPage))
-            .build()
-            .start()
-            .join();
-    }
-}
+ import static org.yaml.snakeyaml.env.EnvScalarConstructor.ENV_FORMAT;
+ import static org.yaml.snakeyaml.env.EnvScalarConstructor.ENV_TAG;
+ import java.util.Collections;
+ import lombok.extern.slf4j.Slf4j;
+ import org.slf4j.Marker;
+ import org.slf4j.MarkerFactory;
+ import org.yaml.snakeyaml.LoaderOptions;
+ import org.yaml.snakeyaml.TypeDescription;
+ import org.yaml.snakeyaml.Yaml;
+ import org.yaml.snakeyaml.env.EnvScalarConstructor;
+ import com.linecorp.armeria.common.SessionProtocol;
+ import com.linecorp.armeria.server.HttpService;
+ import com.linecorp.armeria.server.Server;
+ import com.linecorp.armeria.server.file.FileService;
+ import com.linecorp.armeria.server.file.HttpFile;
+ import com.linecorp.armeria.server.healthcheck.HealthCheckService;
+ 
+ @Slf4j
+ public class ApplicationStartUp {
+     public static void main(String[] args) throws Exception {
+         final Marker startedMarker = MarkerFactory.getMarker("SkyWalking booster UI is running");
+         final Yaml yaml = new Yaml(
+             new EnvScalarConstructor(
```

Comment:
Could you revert all irrelative changes in this file? It seems you reformat the file?